src/MediaHandler.js

```diff
@@ -743,7 +757,7 @@ export default class MediaHandler {
     } = blob;
     const ext = mime.getExtension(blob.contentType) || 'bin';
     let fragment = '';
-    if (blob.meta && blob.meta.width && blob.meta.height) {
+    if (blob.meta && blob.meta.width && blob.meta.height && !blob.contentType?.match(/^video\/[^/]+$/)) {
```

Comment:
add width and height fragment for images only

Reply:
or `&& blob.meta.duration === undefined` ?

Reply:
the duration **should** be available but it doesn't have to: the only information this parser is guaranteed to return is the type `video/something`.